docs/web_reference.rst

```diff
@@ -2904,6 +2905,11 @@ Utilities
                            this flag when being created. This option is not
                            supported on Windows.
 
+   :param bool cancel_handler_on_connection_lost: tells the runner whether to
+                                                  cancel the execution of the
+                                                  handler task if the client
+                                                  connection has been closed.
```

Comment:
I think we probably want to add a section to the advanced docs and link to it here. That section should then highlight the reasons to enable it (save resources, DoS) and highlight the tools/techniques needed to write safe code.

I can probably do it in a separate PR, but if you want to get started that would be great. Looks like Github is starving us of workers right now, so I can't get the tests to run just yet anyway.

Reply:
I still hope that my facts will be taken into account and we will change the default value to `True`.